noxfiles/docker_nox.py

```diff
@@ -30,12 +30,12 @@ def verify_git_tag(session: nox.Session) -> str:
     """
     existing_commit_tag = get_current_tag(existing=True)
     if existing_commit_tag is None:
-        session.error(
+        session.skip(
```

Comment:
handy little method here!

Reply:
yes it is! thank you 🙏 i'd been looking for just that and hadn't found it :)